openapi.yaml

```diff
@@ -0,0 +1,698 @@
+swagger: '2.0'
+info:
+  title: 'Game Curator API'
+  description: 'Learning project API to retrieve and manipulate game review data.'
+  version: 'v1'
+  license:
+    name: GNU Affero General Public License Version 3
+    url: http://www.gnu.org/licenses/agpl-3.0.en.html
+  contact:
+    name: IS Data Architecture Team
+    url: https://is.oregonstate.edu/data-architecture
+    email: isdataarchitecture@oregonstate.edu
+schemes:
+  - http
+host: 'api.oregonstate.edu'
+basePath: /v1
+externalDocs:
+  description: GitHub Repository
+  url: https://github.com/osu-mist/game-curator-api
+produces:
+  - application/json
+paths:
+  /games:
+    parameters:
+      - $ref: '#/parameters/authorization'
+    get:
+      summary: List all games
+      description: Get all games
+      tags:
+        - Games
+      parameters:
+        - $ref: '#/parameters/pageNumber'
+        - $ref: '#/parameters/pageSize'
+        - name: developerId
+          in: query
+          type: integer
+          required: false
+          description: id of developer to return games from
+        - name: name
+          in: query
+          type: string
+          required: false
+          description: name of game to return
+        - name: score_min
+          in: query
+          type: string
+          required: false
+          description: Filter results by minimum review score
+        - name: score_max
+          in: query
+          type: string
+          required: false
+          description: Filter results by maximum review score
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/GameResults'
+        '400':
```

Comment:
When will this method return a 400?

Reply:
added minimum and maximum to scoreMin and scoreMax that will now trigger the 400